packages/react-conformance/src/isConformant.ts

```diff
@@ -1,15 +1,29 @@
-import * as React from 'react';
-import { IsConformantOptions, ConformanceTest } from './types';
+import { TestObject, TestingOptions } from './types';
+import { withCustomConfig } from 'react-docgen-typescript';
+import * as path from 'path';
+import { defaultTests } from './defaultTests';
 
-export function isConformant(
-  Component: React.ComponentType,
-  options: IsConformantOptions,
-  extraTests?: ConformanceTest[],
-) {
-  // const { constructorName, exportedAtTopLevel = true, packageRoot } = options;
-  // if (extraTests) {
-  //   extraTests.forEach((test: ConformanceTest) => {
-  //     test();
-  //   });
-  // }
+export function isConformant(options: TestingOptions, extraTests?: TestObject) {
+  const { componentPath } = options;
+  const tsconfigPath = path.join(process.cwd(), 'tsconfig.json');
+  const parser = withCustomConfig(tsconfigPath, {});
+  const components = parser.parse(componentPath);
+
+  if (components.length === 1) {
```

Comment:
Maybe a good way to handle the multiple components case and verify that the display name is correct would be something like this:
```ts
const mainComponents = components.filter(comp => comp.displayName === testInfo.displayName);
if (mainComponents.length === 1) {
  // do the tests
} else {
  // throw an error
}
```